.github/workflows/manifest-e2e.yaml

```diff
@@ -57,9 +57,19 @@ jobs:
           run_matrix=$run_matrix"]}"
           echo "run_matrix=${run_matrix}" >> $GITHUB_OUTPUT
 
-  manifest-test:
+  mega-image-build:
     needs: job1
     if: always() && ${{ needs.job1.outputs.run_matrix.include.length }} > 0
```

Comment:
If the matrix is empty, the test won't continue. So generally we don't need this line. e.g. https://github.com/opea-project/GenAIComps/actions/runs/9483916020/job/26132175597?pr=156

Reply:
resolved.